dependencies/aws-sdk-cpp/meson.build

```diff
@@ -0,0 +1,51 @@
+# aws s3 dependency file
+#============================================================================
+
+
+
+
+# Look for aws-cpp-sdk.
+#--------------------------------------------------
+if (get_option('with_s3'))
```

Comment:
using parens around single line if statements ins unusual style, for meson, it's more python or ruby-esque than C.

Reply:
I used a set around all statements to try and keep the formatting the same throughout the meson code